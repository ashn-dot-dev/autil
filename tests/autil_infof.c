#define AUTIL_IMPLEMENTATION
#include "../autil.h"

int
main(void)
{
    autil_infof("SOME INFO MESSAGE");
    puts("THIS SHOULD GET PRINTED");
    return EXIT_SUCCESS;
}
