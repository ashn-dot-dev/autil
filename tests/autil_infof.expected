info: SOME INFO MESSAGE
THIS SHOULD GET PRINTED
